composer.json

```diff
@@ -15,21 +15,34 @@
         {
             "name": "Qiang Xue",
             "email": "qiang.xue@gmail.com"
+        },
+        {
+            "name": "Alex Loban",
+            "email": "lav451@gmail.com"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
         "yiisoft/yii2": "~2.0.13",
-        "yiisoft/yii2-bootstrap": "~2.0.0"
+        "yiisoft/yii2-bootstrap4": "1.0.x-dev",
```

Comment:
the debug toolbar should not depend on a bootstrap package. This has been a problem before. It should ship with a fixed version of the bootstrap css it needs.

Reply:
Or with our own CSS. Whatever fits it better...